doc/GUIDE.md

```diff
@@ -577,46 +439,25 @@ default as well).
 
 ## Resolvers and changing your compiler version
 
-Let's explore package sets a bit further. Instead of lts-3.2, let's change our
-`stack.yaml` file to use
-[nightly-2015-08-26](https://www.stackage.org/nightly-2015-08-26). Rerunning
-`stack build` will produce:
+Let's explore package sets a bit further. Instead of lts-10.0, let's change our
+`stack.yaml` file to use [the latest nightly](https://www.stackage.org/nightly). Right now,
+this is currently 2017-12-19 - please see the resolve from the link above to get the latest.
+
+Then, Rerunning `stack build` will produce:
 
 ```
 michael@d30748af6d3d:~/helloworld$ stack build
-Downloaded nightly-2015-08-26 build plan.
-Caching build plan
-stm-2.4.4: configure
-stm-2.4.4: build
-stm-2.4.4: install
-acme-missiles-0.3: configure
-acme-missiles-0.3: build
-acme-missiles-0.3: install
-helloworld-0.1.0.0: configure
-Configuring helloworld-0.1.0.0...
-helloworld-0.1.0.0: build
-Preprocessing library helloworld-0.1.0.0...
-In-place registering helloworld-0.1.0.0...
-Preprocessing executable 'helloworld-exe' for helloworld-0.1.0.0...
-Linking .stack-work/dist/x86_64-linux/Cabal-1.22.4.0/build/helloworld-exe/helloworld-exe ...
-helloworld-0.1.0.0: install
-Installing library in
-/home/michael/helloworld/.stack-work/install/x86_64-linux/nightly-2015-08-26/7.10.2/lib/x86_64-linux-ghc-7.10.2/helloworld-0.1.0.0-6cKaFKQBPsi7wB4XdqRv8w
-Installing executable(s) in
-/home/michael/helloworld/.stack-work/install/x86_64-linux/nightly-2015-08-26/7.10.2/bin
-Registering helloworld-0.1.0.0...
-Completed all 3 actions.
+Downloaded nightly-2017-12-19 build plan.
+# build output ...
 ```
 
 We can also change resolvers on the command line, which can be useful in a
 Continuous Integration (CI) setting, like on Travis. For example:
 
 ```
-michael@d30748af6d3d:~/helloworld$ stack --resolver lts-3.1 build
-Downloaded lts-3.1 build plan.
-Caching build plan
-stm-2.4.4: configure
-# Rest is the same, no point copying it
+michael@d30748af6d3d:~/helloworld$ stack --resolver lts-9.18 build
```

Comment:
Might want to avoid specifying a resolver with a different GHC than the resolver above. It's a little wasteful if they won't end up using that GHC ever.